LDAR_Sim/model_code/methods/deployment/base.py

```diff
@@ -0,0 +1,53 @@
+class sched_crew:
```

Comment:
1. Why is this file simply called "base"?
2. Please add comments where helpful

Reply:
fixed